.github/workflows/release.yml

```
@@ -1,16 +1,6 @@
 name: Release
 
 on:
-  pull_request:
-    branches:
-      - '*'  # must quote since "*" is a YAML reserved character; we want a string
-    paths-ignore:
-      - '.github/workflows/quarto-render.yml'
-      - '_quarto.yml'
-      - 'quarto-materials/*'
-      - '**/.md'
-      - 'tiledb/doxygen/source/*'
-      - 'tiledb/sm/c_api/tiledb_version.h'
```

Suggest to add this as also proposed by @teo-tsirpanis 

```suggestion
  workflow_dispatch:
```

---

Ah, and you beat me to too it.  Very well.